files/poketool/icongra/poke_icon/poke_icon.mk

```diff
@@ -0,0 +1,36 @@
+POKE_ICON_DIR := files/poketool/icongra/poke_icon
+POKE_ICON_NARC := $(POKE_ICON_DIR)/poke_icon.narc
+POKE_ICON_BUILD_DIR := $(POKE_ICON_DIR)/build-poke_icon
+
+POKE_ICON_GFX_FLAGS_ICON := -clobbersize -version101
+POKE_ICON_GFX_FLAGS_PAL := -bitdepth 4
+
+POKE_ICON_PAL_FILES := $(wildcard $(POKE_ICON_DIR)/*.pal)
+POKE_ICON_PAL_OBJS := $(patsubst $(POKE_ICON_DIR)/%.pal,$(POKE_ICON_BUILD_DIR)/%.NCLR,$(POKE_ICON_PAL_FILES))
+POKE_ICON_ANIM_FILES := $(POKE_ICON_DIR)/0_00000001.json $(POKE_ICON_DIR)/0_00000003.json $(POKE_ICON_DIR)/0_00000005.json
+POKE_ICON_ANIM_OBJS := $(patsubst $(POKE_ICON_DIR)/%.json,$(POKE_ICON_BUILD_DIR)/%.NANR,$(POKE_ICON_ANIM_FILES))
+POKE_ICON_CELL_FILES := $(POKE_ICON_DIR)/0_00000002.json $(POKE_ICON_DIR)/0_00000004.json $(POKE_ICON_DIR)/0_00000006.json
+POKE_ICON_CELL_OBJS := $(patsubst $(POKE_ICON_DIR)/%.json,$(POKE_ICON_BUILD_DIR)/%.NCER,$(POKE_ICON_CELL_FILES))
+POKE_ICON_ICON_FILES := $(wildcard $(POKE_ICON_DIR)/*.png)
+POKE_ICON_ICON_OBJS := $(patsubst $(POKE_ICON_DIR)/%.png,$(POKE_ICON_BUILD_DIR)/%.NCGR,$(POKE_ICON_ICON_FILES))
+
+$(POKE_ICON_BUILD_DIR)/%.NCLR: $(POKE_ICON_DIR)/%.pal
+	@mkdir -p $(POKE_ICON_BUILD_DIR)
```

Comment:
why not build in place?
alternatively, shove these in /build

Reply:
`build/` doesn't have any requisite folder structure from what I can see, and building in-place is bad practice. This feels out-of-scope.

Reply:
disagree, building in place is the default, and should be adhered to, changing that should be done separately

Reply:
I don't know that the project can call it a default when some existing targets don't do it (this one is modeled after the target for `pokegra`, which also builds to a target directory). That being said, I can understand wanting to target an initiative; my question then is: what *is* the initiative? Do you want the project to always build in-place, or build to a target folder?

Reply:
ideally in place, everything is already set up for that